definitions/npm/ws_v7.x.x/flow_v0.104.x-/test_v7.x.x.js

```diff
@@ -0,0 +1,382 @@
+// @flow
+
```

Comment:
Please use `describe` and `it` blocks to organize the test file a little more.

Reply:
Thank you for the review. I organised the tests in the commit below.